prompb/remote.proto

```diff
@@ -43,3 +65,12 @@ message QueryResult {
   // Samples within a time series must be ordered by time.
   repeated prometheus.TimeSeries timeseries = 1;
 }
+
+// ChunkedReadResponse is a response when response_type equals STREAMED_XOR_CHUNKS.
+// We strictly stream full series by series. Single frame can contain partion of the single series.
+message ChunkedReadResponse {
+  repeated prometheus.ChunkedSeries chunked_series = 1;
+
+  // This is used when we have more than one ReadRequest.queries.
+  int64 query_index = 2;
```

Comment:
Shouldn't it always be set, even when there's only one query?

Reply:
Wasted 64 bits in most cases, but otherwise yes - let's simplify this and set always.

Reply:
Proto is more efficient than that on the wire.

Reply:
True, agree then.